CHANGELOG.md

```
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [3.0.40](https://github.com/juanjoGonDev/fastypest/compare/v3.0.39...v3.0.40) (2026-07-31)
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)